libraries/engage/tracking/components/CookieConsentModal/connector.js

```diff
@@ -0,0 +1,27 @@
+import { connect } from 'react-redux';
+import { makeGetPrivacyPolicyLink } from '@shopgate/engage/page/selectors';
+import { getIsCookieConsentModalVisible } from '../../selectors/cookieConsent';
+import { acceptAllCookies, acceptRequiredCookies, openPrivacySettings } from '../../actions';
+
+/**
+ * @return {Object} The extended component props.
+ */
+const makeGetStateToProps = () => {
```

Comment:
it's a typo, but in a different way than reported. it needs to be `makeMapStateToProps` - i've corrected it.
it has a different structure than mapStateToProps since a make pattern selector is used here.